### PR TITLE
systemctl: Remove check that linked unit files must contain a "/"

### PR DESCRIPTION
**Pull request by daandemeyer**
This doesn't seem like a useful check, since we make the path absolute right after anyway.

### CODE REVIEW

**Comment by keszybz** [comment]
The idea behind this check is that the user must specify a _path_, and not a filename. Of course there's an overlap between the two concepts, so in other words, we rejects paths that don't contain a slash separator. If a user wants to link a file in a local directory, they should say `./<filename>`. This is because we want to keep the distinction between arguments which are unit names and arguments which are file paths clear. 

Is this removal trying to fix some issue?

**Comment by DaanDeMeyer** [comment]
I was rather surprised by not being able to link a file in the current working directory, but maybe we can just make the error a little clearer without removing the check then, let me update this.

File: src/systemctl/systemctl-enable.c [hunk 1]
```diff
@@ -25,7 +25,7 @@ static int normalize_filenames(char **names) {
 
                         if (!strchr(*u,'/'))
                                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
-                                                       "Link argument does contain at least one directory separator: %s",
+                                                       "Link argument does contain at least one directory separator (try ./%s instead)",
```
Comment (keszybz) [inline src/systemctl/systemctl-enable.c]: hmm, maybe "(try ./%s if file in the current directory was intended)" ?

**Comment by DaanDeMeyer** [comment]
Updated, also made the message multi-line

**Comment by keszybz** [comment]
LGTM.

File: src/systemctl/systemctl-enable.c [hunk 2]
```diff
@@ -25,7 +25,8 @@ static int normalize_filenames(char **names) {
 
                         if (!strchr(*u,'/'))
```
Comment (poettering) [inline src/systemctl/systemctl-enable.c]: might as well add a space behind the `,` while you are at it

**Comment by poettering** [comment]
thanks!

**Comment by keszybz** [comment]
CI passed with the previous commit, and the extra space is unlikely to change the result. Let's merge now.

**Pull request merged**